hw/drivers/chg_ctrl/sgm4056/src/sgm4056.c

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright 2020 Casper Meijn <casper@meijn.net>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "sgm4056/sgm4056.h"
+#include "hal/hal_gpio.h"
+
+#if MYNEWT_VAL(SGM4056_USE_CHARGE_CONTROL)
+
+static int 
+sgm4056_chg_ctrl_get_status(struct charge_control * chg_ctrl, int * status)
```

Comment:
Mynewt code usually does not have space between * and variable even though it's not enforce by style checker as I see.
If you fill confident about force-pushing to you branch you can do it, then you changes will look perfect in git history once merged.
If not just add commit with minor style changes that checker complained about.

Reply:
I fix this. The rebase was more complex then I expected, but I managed to figure it out ;-)